src/tekster/hjelpetekster/banner/banner-overskrift-manuell_nb.txt

```diff
@@ -1 +1 @@
-Registrering av arbeidssøker
\ No newline at end of file
+Start arbeidsrettet oppfølging av sykmeldt bruker
```

Comment:
Denne teksten brukes for både ordinær og sykmeldt registrering, så tenker vi bør utvide med en ny for å dekke behovet. Hva tenker du?

Reply:
Mye kokker på denne nå ass ;-) Men jeg tolker det også på samme måte som deg @JohRus

Reply:
Ok, fikser

Reply:
"Registrering av arbeidsøker" på ordinær?

Reply:
Yes